modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-ibm-storage_{context}"]
+= Setting up {ibm-cloud-title} Object Storage
+
+You can set up {ibm-cloud-title} Object Storage by using the {oc-first}.
+
+.Prerequisites
+
+* You have installed the latest version of `oc`. For more information, see "Getting started with the OpenShift CLI" in _CLI tools_.
+
+* You have installed the latest version of the {ibm-cloud-title} Command Line Interface (AWS CLI). For more information, see "Getting started with the {ibm-cloud-title} CLI" in _{ibm-cloud-title} Docs_.
+
+* You are ready on {ibm-cloud-title}. For more information, see "Choosing a plan and creating an instance" in _{ibm-cloud-title} Docs_.
```

Comment:
Wonder if something more specific than "You are ready on IBM Cloud" could go here. Like, "You have completed the required configuration steps on IBM Cloud" or similar?